components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -0,0 +1,292 @@
+
+| **Column** | **Description** | **Example** |
+| `kbc_project_id` (PK) | Foreign key to the KBC project | `458_eu-central-1_aws` |
+| `snowflake_job_start_at` (PK) | Datetime hour the jobs started (Snowflake jobs/queries are aggregated per hour) | `2019-08-19 06:00:00` |
+| `dwh_size` (PK) | Size of the DWH used | `Medium` |
+| `snowflake_dwh` (PK) | DWH name | `KEBOOLA_PROD` |
+| `snowflake_database` (PK) | DB name | `KEBOOLA_391` |
+| `snowflake_schema` (PK) | Schema name | `WORKSPACE_146192784` |
+| `snowflake_user` (PK) | User running the queries | `KEBOOLA_WORKSPACE_146192784` |
+| `snowflake_job_type` (PK) | Type of the Snowflake job. <br> Possible values: <br> `dwhm` – queries run via DWH manager, `sandbox` – queries run in SQL sandbox, `transformations` – queries run in SQL transformations, `writer` – queries run against Keboola-provisioned DB | `writer` |
+| `snowflake_job_result` (PK) | Result of the queries (`Success`, `Error`) | `Success` |
+| `snowflake_queries` | Number of the queries aggregated by the primary key | `19` |
+| `snowflake_queries_length_s` | Length of the queries in seconds aggregated by the primary key | `205.214000` |
+| `time_credits_used` | Number of the time credits consumed by the queries | `0.91206222224` |
+| `billed_credits_used` | Number of the actually billed credits | `0.91206222224` |
+
+
+### kbc_table_snapshot
+This table shows Storage table snapshots.
+
+| **Column** | **Description** | **Example** | 
+| `table_id` (PK) | Storage table identifier | `in.c-GDU_Management.status` |
+| `kbc_project_id` (PK) | Foreign key to the KBC project | `239_eu-central-1_aws` |
+| `snapshot_date` (PK) | Date of the data snapshot | `2020-07-02` |
+| `created` | Datetime the table was created | `2020-07-02 08:27:03` |
+| `last_import` | Datetime of the last import to the table | `2020-07-02 08:33:39` |
+| `table_name` | Name of the Storage table | `status` |
+| `primary_key` | Primary key of the Storage table | `id` |
+| `is_alias` | Determines if the table is an alias (`true`, `false`). | `true` |
+| `alias_column_sync` | Determines if the alias table should sync all of the columns from the source table (`true`, `false`). | `true` |
+| `source_project_id` | Foreign key to the KBC project the alias is coming from | `20-eu-central-1` |
+| `source_table_id` | Identifier of the source table | `out.c-GDUserManagement.status` |
+| `alias_filter_column` | Column used to filter rows of the source table | `USER_ID` |
+| `alias_filter_operator` | Type of the alias filter operator. Possible values: <br> `eq` -- SQL “IN” <br> `ne` -- SQL “NOT IN” | `eq` |
+| `alias_filter_value` | Value the alias filter is filtering by | `ASNDUH8737D` |
+| `rows` | Number of rows in the table | `8` |
+| `bytes` | Data size of the table in bytes | `2048` |
+
+
+### kbc_usage_metrics_values
+This table shows aggregated values of all metrics that may be part of the contract, for example, consumed credits,
```

Comment:
https://help.keboola.com/management/project/limits/

Reply:
Tohle trochu smrdí. Vůbec není realita, že by podle kontraktů byly v UI vidět limity.
Kdyžtak by to mělo mířit sem: https://help.keboola.com/management/project/limits/#business-limits
Ale zároveň by z dokumentace mělo zmizet to, že najdeš limity v settings.

Reply:
jo tohle smrdí, project/limits by jistě zasloužilo update, ale Milanovi se to nějak nechce updatnout :)

Reply:
zatim teda davam https://help.keboola.com/management/project/limits/#business-limits. Milana poprosim, at to aktualizuje.